catalog/app/containers/Admin/Buckets/Buckets.tsx

```diff
@@ -631,6 +637,37 @@ function BucketFields({ bucket, reindex }: BucketFieldsProps) {
           )}
         </M.Box>
       </M.Accordion>
+      <M.Accordion elevation={0} className={classes.panel}>
+        <M.AccordionSummary
+          expandIcon={<M.Icon>expand_more</M.Icon>}
+          classes={{
+            root: classes.panelSummary,
+            content: classes.panelSummaryContent,
+          }}
+        >
+          <M.Typography variant="h6">File preview options</M.Typography>
+        </M.AccordionSummary>
+        <M.Box className={classes.group} pt={1}>
+          <RF.Field
+            component={Form.Checkbox}
+            type="checkbox"
+            name="browsable"
+            label={
+              <>
+                Enable permissive HTML rendering
```

Comment:
i'm not sure about language here @akarve pls halp
a link to the documentation would be useful too

Reply:
@robnewman can you take a look on language here and in [docs/Catalog/Preview.md](https://github.com/quiltdata/quilt/pull/3198/files#diff-4639943b490f4871476e5c34d2e18e7aaee1ef076a2a4d5b85c09698d6f65a13)

Reply:
I added task to Asana